### PR TITLE
docs: Latest for Cloud-RAD

### PR DESCRIPTION
**Pull request by lesv**
* Initial try at DocFX (still in progress)
* send correct paths to Javadoc / docLava
* Simplify AntRun processing

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)

@tbpg PTAL

### CODE REVIEW

**Comment by lesv** [comment]
@chingor13 @BenWhitehead Is it possible that this is failing because I didn't change a java source file?

```
Run mvn -P lint --quiet --batch-mode checkstyle:check
  mvn -P lint --quiet --batch-mode checkstyle:check
  shell: /bin/bash -e {0}
  env:
    JAVA_HOME_8.0.265_x64: /opt/hostedtoolcache/jdk/8.0.265/x64
    JAVA_HOME: /opt/hostedtoolcache/jdk/8.0.265/x64
    JAVA_HOME_8_0_265_X64: /opt/hostedtoolcache/jdk/8.0.265/x64
Error: No such file or directory
```

**Comment by chingor13** [comment]
No, it's failing because that CI check is templatized, but this repo doesn't have any samples directory.

File: pom.xml [hunk 1]
```diff
@@ -221,7 +225,7 @@
             <dependency>
               <groupId>com.google.googlejavaformat</groupId>
               <artifactId>google-java-format</artifactId>
-              <version>1.7</version>
+              <version>1.9</version>
```
Comment (chingor13) [inline pom.xml:228]: Please don't change this - it requires Java 8+ and will cause the Java 7 build to fail even if it's configured to skip.

File: pom.xml [hunk 2]
```diff
@@ -640,12 +644,59 @@
...
+                  <goal>aggregate</goal>
+                </goals>
+              </execution>
+            </executions>
+            <configuration>
+              <doclet>com.microsoft.doclet.DocFxDoclet</doclet>
+              <docletArtifact>
+                <groupId>com.microsoft</groupId>
+                <artifactId>docfx-doclet</artifactId>
+                <version>1.0-SNAPSHOT</version>
```
Comment (chingor13) [inline pom.xml:682]: Is this snapshot version intentional? It's possible to use Sonatype's nightly repositories, but it's not configured by default (we have an `enable-snapshots` profile that sets up the remote Maven repo).

**Review by chingor13 — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (lesv) [reply pom.xml:228]: I don't recall changing this, but intellij may have.  (Argh)

File: pom.xml (hunk 2 above)
Comment (lesv) [reply pom.xml:682]: I'll put a comment, but this is temporary and NO ONE should be invoking this yet.

**Comment by lesv** [comment]
@chingor13 PTAL

**Review by chingor13 — APPROVED** [review]

**Pull request merged**